MAUI/Rotator/Dots-Customization.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: post
+title: Dots Customization in .NET MAUI Rotator control | Syncfusion
+description: Learn about Dots Customization support in Syncfusion .NET MAUI Rotator (SfRotator) control and more.
+platform: maui 
+control: Rotator 
+documentation: ug
+---
+
+# Dots Customization in .NET MAUI Rotator (SfRotator)
+
+The Rotator control supports customizing dots' border color, selected dots' color, and unselected dots' color.
+
+## DotsBorder Color
+
+The `DotsBorderColor` property is used to customize the color of the dots border in the `SfRotator`.
+
+{% tabs %}
+{% highlight xaml %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Rotator;assembly=Syncfusion.Maui.Rotator"
+                xmlns:local="clr-namespace:Rotator"
+                x:Class="Rotator.Rotator">
+        <ContentPage.BindingContext>
+            <local:RotatorViewModel/>
+        </ContentPage.BindingContext>
+        <ContentPage.Content>
+            <syncfusion:SfRotator x:Name="rotator" 
```

Comment:
Is this code deployed in MAUI?

Reply:
Yes Sir, I have run all the code snippet used.